catalog/schematron/catalog-schematron-plugin/pom.xml

```diff
@@ -69,7 +69,8 @@
                             ddf.services.schematron;version=${project.version}
                         </Export-Package>
                         <Embed-Dependency>
-                            catalog-core-api-impl
+                            catalog-core-api-impl,
+                            Saxon-HE
                         </Embed-Dependency>
                     </instructions>
                 </configuration>
```

Comment:
Can you update the code coverage? I'm sure it's higher than zero now. :smile_cat: